### PR TITLE
Fix get_center method for the jupyter widget and add a get_fov method for both widgets

### PR DESCRIPTION
**Pull request by jsub1**
Part of the work needed to implement issue #200. This uses traitlets to connect the jupyter widget's properties back into the python class.

### CODE REVIEW

File: lib/wwt.js [hunk 1]
```diff
@@ -35,6 +39,18 @@ var WWTView = widgets.DOMWidgetView.extend({
 
         WWTView.__super__.initialize.apply(this, arguments);
 
+        var self = this;
+        setInterval(function () { self.update_view_data(); }, 100);
```
Comment (pkgw) [inline lib/wwt.js]: If I'm reading this correctly, we set up to call `update_view_data()` periodically every 100 ms, right? That's not the end of the world but it's a bit less than ideal — is this definitely necessary? If so, a comment in the code explaining why would be helpful.

File: pywwt/jupyter.py [hunk 2]
```diff
@@ -53,6 +56,16 @@ def _send_msg(self, **kwargs):
     def _serve_file(self, filename, extension=''):
         return serve_file(filename, extension=extension)
 
+    def _get_view_data(self, field):
+        if (field == 'ra'):
```
Comment (pkgw) [inline pywwt/jupyter.py]: You can (read: should) leave out the parentheses here.

**Comment by pkgw** [comment]
Also, the Travis CI failures are due to the "link check" stage of the docs build: [see log here](https://travis-ci.org/WorldWideTelescope/pywwt/jobs/530790254#L1798). That breakage is unrelated to this PR, but if you can update the relevant URLs while you're at it, that would be good to straighten out.

**Comment by jsub1** [comment]
I couldn't figure out a way to do it so that the properties are only updated when the python code requests it. Specifically, I can't figure out a way to block the python code until the properties have been update or found to not need updating on the javascript side without just guessing how long that should/could take and sleeping for that much. If we can work that out, we can remove the periodic calls to update_view_data.

**Comment by pkgw** [comment]
OK, mini literature review on the issue of getting data out of the JavaScript. It seems that this is a known issue:

- [StackOverflow thread from 2017](https://stackoverflow.com/questions/30902898/ipython-notebook-javascript-retrieve-content-from-javascript-variables), no super compelling solution
- [Open notebook issue from 2018](https://github.com/jupyter/notebook/issues/3533), points out existence of [Comm](https://jupyter-notebook.readthedocs.io/en/stable/comms.html) framework, still doesn't seem to solve the blocking issue though
- [Asynchronous Widgets docs from ipywidgets](https://ipywidgets.readthedocs.io/en/stable/examples/Widget%20Asynchronous.html)
- [ipython_blocking] package

[ipython_blocking]: https://github.com/kafonek/ipython_blocking

I *think* that I have a proof-of-concept based on the technique in [ipython_blocking] that works! Basically, you can block the Python code by manually running the Jupyter event queue and storing events until you see one indicating that your operation has completed. I haven't done a detailed implementation, but the basic approach seems functional.

@astrofrog what do you think? The timeout approach clearly has its weaknesses, but it ought to work pretty well. I certainly am a little wary of manually futzing with Jupyter's event loop but it *seems* to work ... and I can certainly imagine that other cases might crop up where we'll need the Python code to talk to the JS code, so a more general framework for doing so might be valuable.

**Comment by pkgw** [comment]
I rebased this on `wwt/master` to hopefully get the CI passing (cf #209).

File: lib/wwt.js (hunk 1 above)
Comment (pkgw) [reply lib/wwt.js]: As per offline discussion with @astrofrog, we're OK with this approach for now. #210 will track the broader issue and potential solutions.

**Pull request merged**